pfcli/deployment.py

```diff
@@ -234,6 +236,9 @@
 )
 deployment_table.add_substitution_rule("Terminated", "[bold]Terminated[/bold]")
 
+deployment_org_table.add_substitution_rule(
+    "Pending", "[bold yellow]Pending[/bold yellow]"
+)
```

Comment:
By the PR https://github.com/friendliai/periflow-cli/pull/165, the "Pending" status is deprecated.

Reply:
Thank you for the remind!